client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -495,9 +472,8 @@ const mapStateToProps = ( state, ownProps ) => {
 		siteIsJetpack,
 		selectedSite: getSelectedSite( state ),
 		storedTitleFormats: getSeoTitleFormatsForSite( getSelectedSite( state ) ),
-		showAdvancedSeo: isAdvancedSeoEligible && isAdvancedSeoSupported,
+		showAdvancedSeo: site && isAdvancedSeoEligible,
```

Comment:
`site` is always truthy, thanks to the `siteSelection` controller. Also, it's confusing that this component gets one instance of `site` from props and another from Redux. There should be only one source of truth, preferrably Redux.

Reply:
Good call, fixed all those in https://github.com/Automattic/wp-calypso/pull/30039/commits/7086328a0b674de741a7e6c33c0616b405609c08

Reply:
Now when `SeoForm` always uses selected site from Redux and ignores the `site` prop, we can also clean up the [place where it's used](https://github.com/Automattic/wp-calypso/blob/6150b68dff86562ce4a1d66f47e10c19bc678894/client/my-sites/site-settings/seo-settings/main.jsx#L39).

Reply:
Ah, of course! Done in https://github.com/Automattic/wp-calypso/pull/30039/commits/5928ab3a7057177f9bca8a2faf349dc2edfc656f